terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -30,6 +43,78 @@ resource "kubernetes_namespace" "namespace" {
   }
 }
 
+# Workload identity[1] lets us map GCP service accounts to Kubernetes service
+# accounts. We need this so that pods can use GCP API, but also AWS APIs like S3
+# via Web Identity Federation. To use the credentials, the container must fetch
+# the authentication token from the instance metadata service. Kubernetes has
+# features for automatically providing a service account token (e.g. via a
+# a mounted volume[2]), but that would be a token for the *Kubernetes* level
+# service account, and not the one we can present to AWS.
+# [1] https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+# [2] https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection
+
+# For each facilitator, we first create a GCP service account.
+resource "google_service_account" "execution_manager" {
+  provider = google-beta
+  # The Account ID must be unique across the whole GCP project, and not just the
+  # namespace. It must also be less than 30 characters, so we can't concatenate
+  # environment and PHA name to get something unique. Instead, we generate a
+  # random string.
+  account_id   = "prio-${random_string.account_id.result}"
```

Comment:
To be unique in the entire GCP project, this identifier must include the environment (e.g., demo vs staging vs prod vs prod-europe), the ingestor name and the peer data share processor. OK, we could mandate that we run one environment per project and earn back some characters there. We would still need to specify constraints on the Apple and Google ingestor names, and names for PHAs, choosing both carefully so that they fit in the limited space we have here. We could do that, but I think that makes an implementation detail of our server everyone's problem, and you'd need to go back to the larger group and champion this proposal. In any case I'd like to move forward with these names; we can always destroy and re-create the service accounts if we pick a new naming scheme.